static/main.js

```diff
@@ -277,14 +277,29 @@ function initializeResetLayoutLink() {
 
 function initPolicies(options) {
     // Ensure old cookies are removed, to avoid user confusion
-
     jsCookie.remove('fs_uid');
     jsCookie.remove('cookieconsent_status');
-    if (options.policies.privacy.enabled &&
-        options.policies.privacy.hash !== jsCookie.get(options.policies.privacy.key)) {
-        $('#privacy').trigger('click', {
-            title: 'New Privacy Policy. Please take a moment to read it',
-        });
+    if (options.policies.privacy.enabled) {
+        if (jsCookie.get(options.policies.privacy.key) == null) {
+            $('#privacy').trigger('click', {
```

Comment:
super nit, here you use `$(selector)` below you use `$(document).find(selector)`. We've always used the former before, just wondering if there's an advantage either way? If not, consistency would be ace :)

Reply:
makes sense, id's should be unique, so .find should be irrelevant

Reply:
Good catch :)